apps/aehttp/test/aehttp_integration_SUITE.erl

```diff
@@ -727,6 +731,15 @@ contract_transactions(_Config) ->
                                fun get_contract_call_compute/1,
                                fun aect_call_tx:new/1, MinerPubkey),
 
+    {ok, 200, #{<<"tx">> := EncodedUnsignedContractCallComputeTx,
+                <<"tx_hash">> := ContractCallComputeTxHash}} =
+        get_contract_call_compute(ComputeCCallEncoded),
+    sign_and_post_tx(EncodedUnsignedContractCallComputeTx),
+
+    % mine a block
+    aecore_suite_utils:mine_blocks(aecore_suite_utils:node_name(?NODE), 2),
+    tx_is_mined_test(MinerAddress, ContractCallComputeTxHash),
```

Comment:
I see contract call tx created with function and arguments (non-encoded call data) was not mined in chain - now it is posted to node and mined in chain.